web_app/api/user.py

```diff
@@ -135,22 +135,48 @@ async def get_user_contract_address(wallet_id: str) -> GetUserContractAddressRes
     response_description="Total amount for all open positions across all users & \
                               Number of unique users in the database.",
 )
-async def get_stats() -> GetStatsResponse:
-    """
-    Retrieves the total amount for open positions and the count of unique users.
-
+async def get_stats() -> GetStatsResponse: 
+    """ 
+    Retrieves the total amount for open positions converted to USDC 
+    and the count of unique users. 
+    
     ### Returns:
-    - total_opened_amount: Sum of amounts for all open positions.
-    - unique_users: Total count of unique users.
+    - total_opened_amount: Sum of amounts for all open positions in USDC. 
+    - unique_users: Total count of unique users. 
     """
     try:
-        total_opened_amount = position_db.get_total_amounts_for_open_positions()
+        # Fetch open positions amounts by token
+        token_amounts = position_db.get_total_amounts_for_open_positions()
+        
+        # Fetch current prices
+        current_prices = await position_db.get_current_prices()
+        
+        # Convert all token amounts to USDC
+        total_opened_amount = Decimal('0')
+        for token, amount in token_amounts.items():
+            # Skip if no price available for the token
+            if token not in current_prices or 'USDC' not in current_prices:
+                logger.warning(f"No price data available for {token}")
+                continue
+            
+            # If the token is USDC, use it directly
+            if token == 'USDC':
+                total_opened_amount += amount
+                continue
+            
+            # Convert other tokens to USDC
+            # Price is typically in USDC per token
+            usdc_price = current_prices[token]
+            usdc_equivalent = amount * Decimal(usdc_price)
+            total_opened_amount += usdc_equivalent
+        
         unique_users = user_db.get_unique_users_count()
+        
         return GetStatsResponse(
-            total_opened_amount=total_opened_amount, unique_users=unique_users
+            total_opened_amount=total_opened_amount, 
+            unique_users=unique_users
         )
-
-    except AttributeError as e:
```

Comment:
Why did you remove this code?

Reply:
It felt redundant that was why.I thought this was closed already that why I am responding now